src/components/ImportExport.vue

```diff
@@ -19,6 +19,8 @@
             outline
             label="Road name"
             clearable
+            v-if="dialog"
```

Comment:
why do we need this?

Reply:
oh, is it so the autofocus behaves properly?

Reply:
[Yeah](https://stackoverflow.com/questions/51472947/vuetifys-autofocus-works-only-on-first-modal-open)